src/main/java/qna/domain/answer/Answer.java

```diff
@@ -29,11 +31,15 @@ public class Answer extends EntityHistory {
     @Column(nullable = false)
     private boolean deleted = false;
 
-    private Long questionId;
+    @ManyToOne
+    @JoinColumn(name = "question_id")
```

Comment:
```suggestion
    @JoinColumn
```

name이 없다면 어떻게 동작하는지 확인해보면 좋겠습니다~

Reply:
동일하게 동작하는걸로 보인느데 맞으려나요